python/llm/src/ipex_llm/transformers/patches.py

```diff
@@ -17,6 +17,8 @@
 
 from typing import List
 from transformers.dynamic_module_utils import get_imports
+from transformers.utils import is_torch_sdpa_available
```

Comment:
What if for the lower `transformers` version without support for `is_torch_sdpa_available`?

Maybe add sth like
```python
def patch_sdpa_available() -> bool:
    if IPEXImporter.is_xpu_version_installed():
        return False
    else:
        try:
            from transformers.utils import is_torch_sdpa_available
            return is_torch_sdpa_available()
        except xxx
            return False 
```